### PR TITLE
Fixed typos and broken links

### PR DESCRIPTION
**Pull request by claire2327**
Fixed some typos and broken links while I was reading the documentation.
Links to all index.md files in the documentation were broken - I know this issue has already been raised and is being fixed. I changed these links by pointing to the folder instead as I'm not sure how the issue will be solved.

### CODE REVIEW

File: admin/getting-started.md [hunk 1]
```diff
@@ -235,9 +235,9 @@ export default class extends Component {
 
 ### Using the Hydra Data Provider Directly with react-admin
 
-By default, the `HydraAdmin` component shipped with API Platform Admin will generate a convenient admin interface for every resources and every properties exposed by the API. But sometimes, you may prefer having full control over the generated admin.
+By default, the `HydraAdmin` component shipped with API Platform Admin will generate a convenient admin interface for every resource and every propertie exposed by the API. But sometimes, you may prefer having full control over the generated admin.
```
Comment (alanpoulain) [inline admin/getting-started.md]:
```suggestion
By default, the `HydraAdmin` component shipped with API Platform Admin will generate a convenient admin interface for every resource and every property exposed by the API. But sometimes, you may prefer having full control over the generated admin.
```

File: schema-generator/index.md [hunk 2]
```diff
@@ -13,46 +13,46 @@ code complies with [PSR](http://www.php-fig.org/) coding standards.
 inheritance (through the `@AbstractSuperclass` annotation).
 * Data validation through [Symfony Validator](http://symfony.com/doc/current/book/validation.html) annotations including
 data type validation, enum support (choices) and check for required properties.
-* Interfaces and [Doctrine `ResolveTargetEntityListener`](http://doctrine-orm.readthedocs.org/en/latest/cookbook/resolve-target-entity-listener.html)
+* Interfaces and [Doctrine `ResolveTargetEntityListener`](https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/cookbook/resolve-target-entity-listener.html)
```
Comment (dunglas) [inline schema-generator/index.md]: latest

File: schema-generator/configuration.md [hunk 3]
```diff
@@ -410,14 +408,14 @@ types:
 
 ## Interfaces and Doctrine Resolve Target Entity Listener
 
-[`ResolveTargetEntityListener`](http://doctrine-orm.readthedocs.org/en/latest/cookbook/resolve-target-entity-listener.html)
+[`ResolveTargetEntityListener`](https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/cookbook/resolve-target-entity-listener.html)
```
[the remark this answers is not in the document]
Comment (alanpoulain) [reply schema-generator/configuration.md]: IMO it should be `current` for ORM, not `latest`.

File: schema-generator/index.md (hunk 2 above)
Comment (alanpoulain) [reply schema-generator/index.md]: current?

File: client-generator/index.md [hunk 4]
```diff
@@ -16,17 +16,17 @@ Client Generator works especially well with APIs built with the [API Platform](h
 
 ## Features
 
-* Generate high-quality ES6:
+* Generates high-quality ES6:
   * list view (with pagination)
   * detail view
   * creation form
-  * editation form
+  * edition form
```
Comment (alanpoulain) [inline client-generator/index.md]: I think someone made this change specifically (I didn't understand it though...).

Comment (alanpoulain) [reply client-generator/index.md]: https://github.com/api-platform/docs/pull/725

Comment (claire2327) [reply client-generator/index.md]: Never heard this word before !

File: core/data-persisters.md [hunk 5]
```diff
@@ -6,16 +6,16 @@ the `@ApiResource` annotation). This instance contains data submitted by the cli
 process](serialization.md).
 
 A data persister using [Doctrine ORM](http://www.doctrine-project.org/projects/orm.html) is included with the library and
-is enabled by default. It is able to persist and delete objects that are also mapped as [Doctrine entities](https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/basic-mapping.html).
+is enabled by default. It is able to persist and delete objects that are also mapped as [Doctrine entities](https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/basic-mapping.html).
```
Comment (alanpoulain) [inline core/data-persisters.md]:
```suggestion
is enabled by default. It is able to persist and delete objects that are also mapped as [Doctrine entities](https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/basic-mapping.html).
```

File: core/identifiers.md [hunk 6]
```diff
@@ -116,9 +116,9 @@ services:
 Your `PersonDataProvider` will now work as expected!
 
 
-## Supported identifiers
+## Supported Identifiers
 
-ApiPlatform supports the following identifier types:
+Api Platform supports the following identifier types:
```
Comment (alanpoulain) [inline core/identifiers.md]:
```suggestion
API Platform supports the following identifier types:
```

File: core/jwt.md [hunk 7]
```diff
@@ -1,17 +1,16 @@
 # JWT Authentication
 
 > [JSON Web Token (JWT)](https://jwt.io/) is a JSON-based open standard ([RFC 7519](https://tools.ietf.org/html/rfc7519)) for creating access tokens that assert some number of claims. For example, a server could generate a token that has the claim "logged in as admin" and provide that to a client. The client could then use that token to prove that he/she is logged in as admin. The tokens are signed by the server's key, so the server is able to verify that the token is legitimate. The tokens are designed to be compact, URL-safe and usable especially in web browser single sign-on (SSO) context.
-
-[Wikipedia](https://en.wikipedia.org/wiki/JSON_Web_Token)
+> - [Wikipedia](https://en.wikipedia.org/wiki/JSON_Web_Token)
```
Comment (alanpoulain) [inline core/jwt.md]: Are you sure it will work as expected?

File: core/jwt.md [hunk 8]
```diff
@@ -78,13 +77,13 @@ api_platform:
...
 
-And the "Authorize" button will automatically appear in Swagger UI.
+The "Authorize" button will automatically appear in Swagger UI.
 
 ![Screenshot of API Platform with Authorize button](images/JWTAuthorizeButton.png)
 
 ### Adding a New API Key
 
-All you have to do is configuring the API key in the `value` field.
+All you have to do is configure the API key in the `value` field.
```
Comment (alanpoulain) [inline core/jwt.md]: Not sure about this change :thinking:

File: core/performance.md [hunk 9]
```diff
@@ -170,16 +170,15 @@ database driver.
 
 ### Eager Loading
 
-By default Doctrine comes with [lazy loading](http://doctrine-orm.readthedocs.io/en/latest/reference/working-with-objects.html#by-lazy-loading).
-Usually a killer time-saving feature and also a performance killer with large applications.
+By default Doctrine comes with [lazy loading](https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/working-with-objects.html#by-lazy-loading) - usually a killer time-saving feature but also a performance killer with large applications.
 
-Fortunately, Doctrine proposes another approach to remedy this problem: [eager loading](http://doctrine-orm.readthedocs.io/en/latest/reference/working-with-objects.html#by-eager-loading).
+Fortunately, Doctrine offers another approach to solve this problem: [eager loading](https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/working-with-objects.html#by-eager-loading).
```
Comment (alanpoulain) [inline core/performance.md]:
```suggestion
Fortunately, Doctrine offers another approach to solve this problem: [eager loading](https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/working-with-objects.html#by-eager-loading).
```

File: core/serialization.md [hunk 10]
```diff
@@ -296,10 +296,10 @@ The generated JSON using previous settings is below:
 ```
 
 In order to optimize such embedded relations, the default Doctrine data provider will automatically join entities on relations
-marked as [`EAGER`](http://doctrine-orm.readthedocs.io/projects/doctrine-orm/en/latest/reference/annotations-reference.html#manytoone).
+marked as [`EAGER`](https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/annotations-reference.html#manytoone).
```
Comment (alanpoulain) [inline core/serialization.md]:
```suggestion
marked as [`EAGER`](https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/annotations-reference.html#manytoone).
```

File: deployment/traefik.md [hunk 11]
```diff
@@ -21,9 +21,9 @@ services:
     labels: 
       - "traefik.frontend.rule=Host:api.localhost"
 ``` 
-The api DNS will be specified with `traefik.frontend.rule=Host:your.host` (here api.localhost)  
+The Api DNS will be specified with `traefik.frontend.rule=Host:your.host` (here api.localhost)
```
Comment (alanpoulain) [inline deployment/traefik.md]:
```suggestion
The API DNS will be specified with `traefik.frontend.rule=Host:your.host` (here api.localhost)  
```

File: distribution/index.md [hunk 12]
```diff
@@ -488,11 +488,11 @@ A URL is a valid IRI, and it's what API Platform uses. The `@id` property of eve
...
-(e.g. to reduce the number of HTTP requests). You can even [let the client selecting only the properties it needs](../core/filters.md#property-filter).
+(e.g. to reduce the number of HTTP requests). You can even [let the client select only the properties it needs](../core/filters.md#property-filter).
 
 The other interesting thing is how API Platform handles dates (the `publicationDate` property). API Platform understands
 [any date format supported by PHP](http://php.net/manual/en/datetime.formats.date.php). In production we strongly recommend
-to use the format specified by the [RFC 3339](http://tools.ietf.org/html/rfc3339), but, as you can see, most common formats
+using the format specified by the [RFC 3339](http://tools.ietf.org/html/rfc3339), but, as you can see, most common formats
 including `September 21, 2016` can be used.
 
 To summarize, if you want to expose any entity you just have to:
```
Comment (alanpoulain) [inline distribution/index.md]: We should remove the "just have".

File: distribution/index.md [hunk 13]
```diff
@@ -501,7 +501,7 @@ To summarize, if you want to expose any entity you just have to:
 2. If you use Doctrine, map it with the database
 3. Mark it with the `@ApiPlatform\Core\Annotation\ApiResource` annotation
 
-How can it be more easy?!
+Could it be any easier?!
```
Comment (alanpoulain) [inline distribution/index.md]: Please remove this sentence.

Comment (dunglas) [reply distribution/index.md]: Why?!

File: core/jwt.md (hunk 8 above)
Comment (claire2327) [reply core/jwt.md]: For instance:
https://fr.wikipedia.org/wiki/All_I_Have_to_Do_Is_Dream

File: client-generator/index.md [hunk 14]
```diff
@@ -1,6 +1,6 @@
 # The API Platform Client Generator
 
-Client Generator is the fastest way to scaffold fully featured webapps and native mobile apps from APIs supporting the [Hydra](http://www.hydra-cg.com/) format.
+Client Generator is the fastest way to scaffold fully-featured webapps and native mobile apps from APIs supporting the [Hydra](http://www.hydra-cg.com/) format.
```
[the remark this answers is not in the document]
Comment (claire2327) [reply client-generator/index.md]: "No hyphen is necessary in phrasal adjectives with an adverb ending in -ly, unless the -ly adverb is part of a longer phrase"
You're right :)

**Comment by soyuka** [comment]
@pborreli you have found yourself a rival :smile_cat:

**Comment by pborreli** [comment]
@soyuka a successor, I quit 😄

**Comment by alanpoulain** [comment]
Thank you very much @claire2327!
